### PR TITLE
Task-50800: Implement updateMetadata service API

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/metadata/MetadataServiceImpl.java [hunk 1]
```diff
@@ -80,6 +80,24 @@ public Metadata createMetadata(Metadata metadata, long userIdentityId) {
     }
     return metadata;
   }
+
+  @Override
+  public Metadata updateMetadata(Metadata metadata, long userIdentityId) {
```
Comment (azayati) [inline component/core/src/main/java/org/exoplatform/social/core/metadata/MetadataServiceImpl.java:85]: userIdentityId not needed IMHO

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: component/core/src/main/java/org/exoplatform/social/core/metadata/MetadataServiceImpl.java (hunk 1 above)
Comment (hassenelkamel-exo) [reply component/core/src/main/java/org/exoplatform/social/core/metadata/MetadataServiceImpl.java:85]: used in broadcast event

Comment (azayati) [reply component/core/src/main/java/org/exoplatform/social/core/metadata/MetadataServiceImpl.java:85]: ok we can keep it

**Review by azayati — APPROVED** [review]
LGTM after QA validation

**Pull request merged**